.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[env]
+POSTGRESQL_VERSION = "15"
```

Comment:
oh, cool, I didn't know you could do this.

Reply:
I also only learned this today :O